server/src/models/internship.ts

```diff
@@ -124,6 +128,10 @@ const requiredFields = [
   "workingHoursPerWeek",
   "supervisor.fullName",
   "supervisor.emailAddress",
+  "lsfEctsProofPdf",
+  "locationJustificationPdf",
+  "contractPdf",
```

Comment:
Das funktioniert leider nicht so einfach, weil die alle zwar existieren, aber wir checken müssen, ob auch Events drin sind, die die Pfade setzen. Solange das Model zwar existiert, aber kein Dateipfad vorhanden ist, ist's ja noch nicht gültig.

Reply:
Aha! Ja schade. Und nu?

Reply:
Fixed in 478cdcf4dbd4c59842647efc33afb7af46adbd25